apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationProjects.jsx

```diff
@@ -105,13 +111,7 @@ export default class ModerationProjects extends Component {
                     <div>
                       <span className="text-muted">{byText}{item.organisation}</span>
                       <h3 className="tile__title mb-2">
-                        <a
-                          href={item.moderation_detail_url}
-                          className="tile__link"
-                        >
-                          {item.num_unread_comments > 0 && <span className="text-info">• <span className="visually-hidden">{hasUnReadComments}</span></span>}
-                          {item.title}
-                        </a>
+                        {item.title}
```

Comment:
one thing I noticed is that it is now read differently by the Screenreader (at least the chrome one). It doesn't read the title anymore when  tabbing through (so you need an extra step to read it). Not sure if that's an issue though. @philli-m any thoughts?

Reply:
I can check on JAWS and NVDA as they are the most used

Reply:
@khamui I used the one the chrome screen reader (which probably isn't very good). I can check with the other ones later.

Reply:
NVDA only reads blank, I guess that's not ideal

Reply:
@goapunk @philli-m having the visually hidden h3 as a child worked finally..